core/src/main/java/io/questdb/griffin/engine/groupby/vect/GroupByRecordCursorFactory.java

```diff
@@ -438,17 +438,25 @@ public long getLong(int col) {
 
             @Override
             public void getLong256(int col, CharSink sink) {
-
+                Long256Impl v = (Long256Impl) getLong256A(col);
+                v.toSink(sink);
             }
 
             @Override
             public Long256 getLong256A(int col) {
-                return null;
+                final long offset = getValueOffset(col);
+                final long l0 = Unsafe.getUnsafe().getLong(offset);
+                final long l1 = Unsafe.getUnsafe().getLong(offset + Long.BYTES);
+                final long l2 = Unsafe.getUnsafe().getLong(offset + 2 * Long.BYTES);
+                final long l3 = Unsafe.getUnsafe().getLong(offset + 3 + Long.BYTES);
+                Long256Impl res = new Long256Impl();
```

Comment:
We should not be allocating on execution path. Typical impl is to have two `Long256Impl` per record, e.g. A and B. This also requires a test, where result of this record implementation can be ordered correctly. E.g. `order by long256`